spec/enumerable_spec.rb

```diff
@@ -0,0 +1,168 @@
+require '../enumerables.rb'
```

Comment:
- [ ] Your tests are not running. Please use require_relative instead of require.

> "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."

![image](https://user-images.githubusercontent.com/53256139/91751301-3676f000-eb8a-11ea-9e32-3b643ec002ab.png)